.cargo/config

```diff
@@ -0,0 +1,2 @@
+[target.x86_64-pc-windows-msvc]
+rustflags = ["-C", "link-arg=/STACK:8388608"]
```

Comment:
I'm not sure what to do about this either, it's required in a project that uses swiftnav-rs and here as well, apparently this is by design for how rust links with native libraries:

https://doc.rust-lang.org/beta/unstable-book/language-features/link-args.html

Reply:
Is is something in our code that requires the 8 MB stack size specifically?

Reply:
It would be nice to understand this change more, but during compilation things will crash with a `thread 'main' has overflowed its stack` error otherwise-- I think 4MiB worked, 8MiB was a "just in case value".

LSN on windows uses the stack by default for allocation-- but this is a runtime thing-- I don't think it would causes issues until something was actually invoked.  Nevertheless I tried to enable heap allocation with some compiler flags and it didn't seem to affect this issue.

This made me think it might be a huge table somewhere that gets initialize as part of setting up globals, like the geoid grid... I tried disabling the compilation of the geoid grid but that didn't help.

Didn't try anything else past this though.

Reply:
You're probably right. I'm cool if this is a workaround, maybe we can investigate it later.

Reply:
Ok, this is definitely a workaround for now, at some point we can (hopefully) revisit and see if we can track down what's requiring such a large stack